powerline/bindings/tmux/__init__.py

```diff
@@ -76,6 +76,8 @@ def get_tmux_version(pl):
 	version_string = get_tmux_output(pl, '-V')
 	_, version_string = version_string.split(' ')
 	version_string = version_string.strip()
+	if version_string == 'master':
+		return TmuxVersionInfo(math.inf, 0, version_string)
```

Comment:
And where is the `math` import? Also `math` module does not have `inf` attribute. Just use `float('inf')`, your variant is completely broken.